test/consensus/join_during_creation.js

```diff
@@ -161,17 +121,20 @@ contract('Consensus::joinDuringCreation', (accounts) => {
         joinParams.withdrawalAddress,
         'Withdrawal address not set in spy reputation contract',
       );
-    });
-
-    it('should call join function of core contract with correct params', async () => {
-      await consensusUtil.joinDuringCreation(consensus, joinParams);
 
       const spyValidator = await core.spyValidator.call();
       assert.strictEqual(
         spyValidator,
         joinParams.txOptions.from,
         'Validator address not set in spy core contract',
       );
+
+      const spyCoreFromConsensusGateway = await consensusGateway.spyCore.call();
+      assert.strictEqual(
+        spyCoreFromConsensusGateway,
+        core.address,
+        'Core address not set in spy consensus gateway contract',
+      );
```

Comment:
Please assert that all the spy params are called. Or add a todo/task note.

Reply:
Done!